nevergrad/optimization/base.py

```diff
@@ -512,7 +513,7 @@ def minimize(
                 while self._finished_jobs:
                     x, job = self._finished_jobs[0]
                     self.tell(x, job.result())
-                    self._finished_jobs.popleft()  # remove it after the tell to make sure it was indeed "told" (in case of interruption)
+                    self._archived_jobs.append(self._finished_jobs.popleft())  # remove it after the tell to make sure it was indeed "told" (in case of interruption)
```

Comment:
what is the aim here?
I'm afraid this way will cause the memory to explode in some cases (very long local trainings for instance) :s for the point archive I've needed to implement a pruning every now and then (which does not satisfy me either :s)

Reply:
Then `_archived_jobs` could be a deque with fixed length to prevent that, what do you think?

The goal is to retrieve the jobs after training, for instance using slurm jobs, I want to be able to retrieve all their ids so that I can do other stuff with it (e.g. with their stdout).

Reply:
my prefered way would be to wrap the executor, I don't think that would generalize well to other type of executor than the slurm one

Reply:
Ok then I'll close the PR